versioned_docs/version-23.04/alerts-notifications/resources-status.md

```diff
@@ -175,7 +175,7 @@ in one or several fields.
 - **service_group**: search for services that belong to a service group
 - **service_severity**: search for services according to the name of their severity
 - **service_severity_level**: search for services according to the level of their severity
-- **s.description**: search only in the description of the service (i.e. its name in column **Resource**)
+- **s.description**: search only in the name of the service (shown in column **Resource**)
```

Comment:
While it is often displayed and refered as "name", it actually looks for the service description field in its configuration. (hence the name description)
I think we should keep "description" or maybe change to "description field"